utils/nctl/sh/scenarios/configs/swap_validator_set.config.toml.override

```diff
@@ -0,0 +1,2 @@
+[binary_port_server]
+allow_request_get_all_values = true
```

Comment:
Wouldn't it be easier to just set it in `resources/local/config.toml`? Or is there a reason not to do that?

Reply:
Yes, probably easier to add it there. Although in `local/config.toml` it's disabled specifically by default intentionally.

Reply:
I mean, I understand why it's disabled in the production config, but local config is only used for tests, isn't it?

Reply:
@rafal-ch any reason why it was disabled in the local config intentionally?

Reply:
None that I know of. Maybe just for the sake of keeping these two configs as much in sync as possible.

Reply:
@fizyk20 set it to false in the local config as suggested here: https://github.com/casper-network/casper-node/pull/4603/commits/b2763eca4b6fb25a8bef2ddf9231f348f5c83509